gui/mupdf.watch.py

```diff
@@ -0,0 +1,19 @@
+from urllib import request
+import json
+import re
+import datetime
+
+data = request.urlopen(request.Request('https://www.mupdf.com/downloads/index.html')).read().decode('utf-8')
+						
+matches =re.findall(r'>mupdf-([0-9\.]+)-windows.zip<', data)
+data = request.urlopen(request.Request('https://www.mupdf.com/release_history.html')).read().decode('utf-8')
+releases = []
+for amatch in matches:
+  date = datetime.datetime.today().date().isoformat()
+  secondMatch = re.search(rf'MuPDF {amatch} \((....-..-..)\)',data)
+  if secondMatch:
+    date = secondMatch[1]
+
+  releases.append({'version': amatch,'released': date })
+    
+releases
```

Comment:
Suggested reformatting to align more with style of other `.py.watch` files in the repository:

```python
import re
from urllib import request
from datetime import datetime

matches = re.findall(r'>mupdf-([0-9\.]+)-windows.zip<', request.urlopen(request.Request('https://www.mupdf.com/downloads/index.html')).read().decode('utf-8'))
release_history = request.urlopen(request.Request('https://www.mupdf.com/release_history.html')).read().decode('utf-8')
releases = []
for match in matches:
  release_match = re.search(rf'MuPDF {match} \((....-..-..)\)', release_history)
  releases.append({
    'version': match,
    'released': release_match[1] if release_match else datetime.today().date().isoformat()
  })
```

Reply:
I have changed the style of the script
let me know if it is still lacking